src/Storefront/Theme/ThemeMergedConfigBuilder.php

```diff
@@ -50,6 +50,7 @@ public function getPlainThemeConfiguration(string $themeId, Context $context): a
 
         $themes = $this->themeRepository->search($criteria, $context)->getEntities();
 
+        /** @var ThemeEntity $theme */
```

Comment:
please never use `@var` annotations :slightly_smiling_face:

Reply:
Faster than the "review" state 😄 Fixed. (I hope)